app/vmselect/netstorage/tmp_blocks_file.go

```diff
@@ -85,7 +85,12 @@ func (addr tmpBlockAddr) String() string {
 	return fmt.Sprintf("offset %d, size %d", addr.offset, addr.size)
 }
 
-var tmpBlocksFilesCreated = metrics.NewCounter(`vm_tmp_blocks_files_created_total`)
+var (
+	tmpBlocksFilesCreated = metrics.NewCounter(`vm_tmp_blocks_files_created_total`)
+	_ = metrics.NewGauge(`vm_tmp_blocks_files_directory_free_space`, func() float64 {
```

Comment:
Could you add `_bytes` suffix to the metric name according to [Prometheus naming convention for metric names](https://prometheus.io/docs/practices/naming/#metric-names)?

Reply:
Done